README.md

```diff
@@ -2,15 +2,18 @@
 *Bot lists must meet these minimum requirements to be considered a bot list.*
 
 ---
-
-
-1. Your bot list must have a respectable TLD. (.tk does not count)
-2. Your bot list can **NOT** be hosted on glitch.
-3. Your bot list should not display all the listed bots on the front page.
-4. Your bot list must be usable. No broken links/buttons.
-5. Your bot list must have consistent uptime. (Minor downtime for fixing a problem is fine)
-
-
+Your bot list...
+1. must have a respectable TLD.
```

Comment:
Could we just make this point that it must have its own custom domain with a respectable (paid) TLD?

That covers the glitch (etc) subdomain point and the tk point?

Reply:
Not everyone is aware that a .tk is not seen as a "respectable" domain.
That paid part could clear it up but I still think mentioning the .tk part should stay.

Reply:
There are many other free TLDs alike to tk. I don’t believe we should list tk as then users may cite that other free TLDs aren’t listed as a defence for using them.